scripts/docs-build.sh

```diff
@@ -11,9 +11,14 @@ yarn install --frozen-lockfile
 yarn build
 cd ..
 
+cd cursorless-nx
+npm ci
+npx nx export cursorless-org
+cd ..
+
 # Since baseUrl in Docusaurus is /docs, for links within our website
 # to work correctly we need to serve /website-root with /docs subfolder
 # containting the build
-rm -rf website-root/docs
-mkdir -p website-root/docs
-cp -r website/build/* website-root/docs
+out_dir=cursorless-nx/dist/apps/cursorless-org/exported/docs
```

Comment:
move nx output somewhere else first so as not to mess with output that it should own